src/syft/util.py

```diff
@@ -122,6 +122,21 @@ def aggressive_set_attr(obj: object, name: str, attr: object) -> None:
         curse(obj, name, attr)
 
 
+@syft_decorator(typechecking=True)
+def aggressive_set_particular_attr(obj: object, name: str, attr: object) -> None:
```

Comment:
This looks like its just doing the same as `aggressive_set_attr`?
```
@syft_decorator(typechecking=True)
def aggressive_set_attr(obj: object, name: str, attr: object) -> None:
    """Different objects prefer different types of monkeypatching - try them all"""

    try:
        setattr(obj, name, attr)
    except Exception:
        curse(obj, name, attr)
```

Reply:
Yes, almost. But it handle some particular exception, other exceptions will still be throw. Anyway, I am deleting this function.